bitcoin/src/electrs.rs

```diff
@@ -10,10 +11,29 @@ async fn get(base_path: &str, url: &str) -> Result<String, Error> {
     }
 }
 
+// the esplora_btc_api lib breaks on these two because it tries to serde_json::from_str on a plain string value
 pub async fn get_tx_hex(base_path: &str, tx_id: &str) -> Result<String, Error> {
     get(base_path, &format!("tx/{}/hex", tx_id)).await
 }
 
 pub async fn get_tx_merkle_block_proof(base_path: &str, tx_id: &str) -> Result<String, Error> {
     get(base_path, &format!("tx/{}/merkleblock-proof", tx_id)).await
 }
+
+// and it doesn't currently support the paged api call
+pub async fn get_address_tx_history_full(base_path: &str, address: &str) -> Result<Vec<Transaction>, Error> {
+    let mut last_seen_txid = "".to_owned();
+    let mut ret = Vec::<Transaction>::default();
+    loop {
+        let mut transactions: Vec<Transaction> =
+            serde_json::from_str(&get(base_path, &format!("address/{}/txs/chain/{}", address, last_seen_txid)).await?)?;
```

Comment:
Nit: shouldn't the trailing `/` be omitted for the first page? The api specification states `GET /address/:address/txs/chain[/:last_seen_txid]`. I guess you tested so it seems like it doesn't matter

Reply:
Works just fine, and this way it doesn't need special-casing.